### DISCUSSION TITLE
basic_file_sink size limit

### DISCUSSION THREAD

**Discussion by dradic2** [gabime/spdlog#2421] [Q&A] [answered]
Hello there,

I have a situation where I am writing to a basic_file_sink for a longer period of time. In the beginning all seems to work well, but after some time the file size falls to 0 bytes.
My assumption is that there is too much data and that at some point the file becomes corrupted.

What is the file size limit for the basic_file_sink?

**Comment by gabime** [comment, marked as answer]
There is no limit. Perhaps it run out of storage space.

**Reply by dradic2** [reply]
Thank you for the answer! I'll look into it.
